src/main/java/codesquad/springcafe/article/ArticleController.java

```diff
@@ -1,57 +1,81 @@
 package codesquad.springcafe.article;
 
+import codesquad.springcafe.article.dto.ArticleUpdateRequestDto;
+import jakarta.servlet.http.HttpSession;
+import jakarta.validation.Valid;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.ModelAttribute;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-
-import java.time.LocalDateTime;
-import java.util.Collection;
+import org.springframework.web.bind.annotation.*;
 
 @Controller
 public class ArticleController {
     private final Logger log = LoggerFactory.getLogger(getClass());
-    private final ArticleDao articleDao;
+    private final ArticleService service;
 
-    public ArticleController(ArticleDao articleDao) {
-        this.articleDao = articleDao;
+    public ArticleController(ArticleService service) {
+        this.service = service;
     }
 
     // 아티클 id를 가지고 해당 아티클 보여주기 없으면 기본 홈페이지 보여주기
     @GetMapping("/article/{id}")
-    public String showArticle(@PathVariable int id, Model model) {
-        return articleDao.findBy(id)
+    public String showArticle(@PathVariable int id, Model model, HttpSession session) {
+        String value = (String) session.getAttribute("sessionUserId");
```

Comment:
```suggestion
        String sessionUserId = (String) session.getAttribute("sessionUserId");
```

명확하게 의미있는 변수명을 사용하는 게 더 좋을 것 같습니다.